test/suites/integration/expiration_test.go

```diff
@@ -61,10 +61,19 @@ var _ = Describe("Expiration", func() {
 		node := env.Monitor.CreatedNodes()[0]
 		env.Monitor.Reset()
 
-		// Eventually expect the node to be gone and a new one to come up
-		env.EventuallyExpectNotFound(node)
```

Comment:
Why isn't it enough to just wait?

Reply:
Because we aren't fast enough. If we wait to update the provisioner after the node is gone, the deprovisioner already gets into another loop of deprovisioning before it sees the provisioner update